legal-api/src/legal_api/models/business.py

```diff
@@ -786,6 +786,23 @@ def is_pending_amalgamating_business(cls, business_identifier):
                    ).one_or_none()
         return filing
 
+    @classmethod
+    def generate_numbered_legal_name(cls, legal_type, identifier):
+        """Generate legal name for a numbered business."""
+        if legal_type not in Business.BUSINESSES:
+            return None
+
+        numbered_legal_name_suffix = Business.BUSINESSES[legal_type]['numberedLegalNameSuffix']
+        numbered_legal_name_prefix = ''
+        if legal_type in (Business.LegalTypes.BCOMP_CONTINUE_IN.value,
+                          Business.LegalTypes.ULC_CONTINUE_IN.value,
+                          Business.LegalTypes.CCC_CONTINUE_IN.value,
+                          Business.LegalTypes.CONTINUE_IN.value):
+            numbered_legal_name_prefix = identifier[1:]
+        else:
+            numbered_legal_name_prefix = identifier[2:]
+        return f'{numbered_legal_name_prefix} {numbered_legal_name_suffix}'
```

Comment:
Isn't there already a similar method for BC/BEN/CC/ULC?

Reply:
Oh, is this converting "C1234567" to "1234567 B.C. Ltd.", for example?

Reply:
> Isn't there already a similar method for BC/BEN/CC/ULC?

Yes. We have the same code in filer. Planning to replace filer code with this func